provisioning/transport/mqtt/src/ProvisioningChannelHandlerAdapter.cs

```diff
@@ -533,5 +541,15 @@ private ushort GetCurrentPacketId()
                 return (ushort)Volatile.Read(ref _packetId);
             }
         }
+
+        private static int jitterMax = 5;
+        private static int jitterMin = 0;
+        private static TimeSpan GenerateDelayWithJitterForRetry(TimeSpan defaultDelay)
```

Comment:
Same as above: please keep all common code in `common\src`

Reply:
Sure thing